guides/development-workflow.md

```diff
@@ -461,26 +445,32 @@ story and the acceptance criteria that should be met.
 
 A good pull request should:
 
-- be created by the authoring developer
-- meet all of the acceptance criteria on the associated story
-- focus on the single problem at hand, not doing so will usually increase the
-  time taken to get it merged
-- include appropriate detail to assist the reviewer, this includes links to
-  relevant content and a duplication of anything of note in the commit
-  descriptions
-- update the [changelog](#tracking-changes) if it's a significant change
-
-```
-# Poor
-A new registration form and fix a bug with contacts
-
-# Great
-[123456789] New users can now register for an account
-
-- This has been rate limited, would like your feedback on the values I've chosen
-- Passwords are encrypted using BCrypt, so I've taken this opportunity to
-  increase the default rounds from 8 to 12
-```
+- describe the problem without assuming the reader has existing context
```

Comment:
Related: should link to the story card (eg in Trello) for extra context and history, but should provide everything within the PR description to understand the change (ie linking to Trello is not enough)